lib/reporter.js

```diff
@@ -84,7 +84,8 @@ class CucumberReporter {
         case Cucumber.Status.PENDING:
         case Cucumber.Status.SKIPPED:
         case Cucumber.Status.AMBIGUOUS:
-            e = 'pending'
+            e = 'skipped'
```

Comment:
if the allure reporter is the only reporter that has problems with it why not just fixing it there?